meshroom/ui/components/filepath.py

```diff
@@ -98,3 +99,17 @@ def globFirst(self, path):
     def fileSizeMB(self, path):
         """ Returns the file size in MB. """
         return QFileInfo(self.asStr(path)).size() / (1024*1024)
+
+    @Slot(str, result="QVariantList")
+    @Slot(str, str, result="QVariantList")
+    def getFilenamesFromFolder(self, folder_path: str, extension: str = None):
```

Comment:
```suggestion
    def getFilenamesFromFolder(self, folderPath: str, extension: str = None):
```

`folder_path` respects snake_case which is usually what's expected in Python... but no what we're using here, starting with the method's name.